docker/wiremock/__files/mockResponse.example.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
Is there any use for that file? I'd easily delete it seeing it is never used. Ideally if script that generates mock response used something like that as a template. That would make it way clearer than modifying JSON objects through 'jq' pipes.

Reply:
Just put it to show how's _mockResponse.json_ supposed to look like. Removed and added _userTemplate.json_ as suggested.